src/Illuminate/Container/BoundMethod.php

```diff
@@ -105,18 +106,25 @@ protected static function normalizeMethod($callback)
      *
      * @param  \Illuminate\Container\Container  $container
      * @param  callable|string  $callback
-     * @param  array  $parameters
+     * @param  array $inputData
      * @return array
+     * @throws \ReflectionException
      */
-    protected static function getMethodDependencies($container, $callback, array $parameters = [])
+    protected static function getMethodDependencies($container, $callback, array $inputData = []) : array
```

Comment:
why did you change `$parameters` to `$inputData`? Cant the name stay as it was to reduce unnecessary changes?

Reply:
of course why not ? the term 'parameter' was overloaded here and it had 2 meanings.
You know it confused me at first, which $parameter refers to the actual user input and which one is the method signature declaration. So I wanted to make it crystal clear